plugins/Ecommerce/Controller.php

```diff
@@ -97,10 +96,19 @@ public function getConversionsOverview()
     public function getEcommerceLog($fetch = false)
     {
         $saveGET = $_GET;
-        $_GET['segment'] = urlencode('visitEcommerceStatus!=none');
+        $originalQuery = $_SERVER['QUERY_STRING'];
+
+        if (!empty($_GET['segment'])) {
+            $_GET['segment'] = $_GET['segment'] . ';' . 'visitEcommerceStatus!=none';
+        } else {
+            $_GET['segment'] = 'visitEcommerceStatus!=none';
+        }
+        $_SERVER['QUERY_STRING'] = http_build_query($_GET);
```

Comment:
You'll need to use Http::buildQuery() for this as there need to be 2 more params set. Otherwise it won't work on some systems (result is different on some servers)